.github/workflows/build.yml

```diff
@@ -108,11 +104,16 @@ jobs:
       matrix:
         scenario:
           - default
-    runs-on: ubuntu-latest
+    # TODO: We would prefer to run this job on ubuntu-latest, but
+    # Amazon Linux 2 has such an ancient version of SystemD (219) that
+    # it can't cope with cgroups v2.  We should revert to
+    # ubuntu-latest once cisagov/skeleton-ansible-role#131 is
+    # approved, merged, and pulled into this repo via Lineage.
```

Comment:
You can skip this if you disagree but thoughts on rephrasing this bit? It reads like 131 is a pull request when it is an issue. Something like "once the change described in cisagov/skeleton-ansible-role#131 is implemented" or whatever you think would be appropriate.

Reply:
I don't disagree, but at this point @dav3r and I have made the exact same change to quite a few repos.  Can you live with this slight inaccuracy for now, since it will be going away in every repo where it appears in the near future (as soon as Amazon Linux 202[23] is officially released)?

Reply:
I will tolerate this minor inaccuracy for the sake of moving forward. Apologies for throwing sand in the gears.

Reply:
No worries.  Your perfectionism is also a blessing.  Thank you for your understanding.